docs/user-guide/configuration/jobconfiguration.md

```diff
@@ -111,15 +115,63 @@ jobs:
         image: node:6
         steps:
             - init: npm install
+            - pretest: npm lint
             - test: npm test
     main2:
         requires: [main]
         image: node:8
         steps:
-            - greet: echo hello
+            - test: echo Skipping test
+
+```
+
+## Merge shared steps
```

Comment:
`mergeSharedSteps` is applicable only when using template

Reply:
removed